downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -113,8 +113,31 @@ It is now possible to provide custom formatting configurations to the Ballerina
 
 #### OpenAPI tool
 
+- Integrated OpenAPI client generation to `bal build` command, in here user can provide the openapi tool config details in to the Ballerina.toml file and generate the client using `bal build` command.
+Example:
+```ballerina
+[[tool.openapi]]
+id = "dBclient"
+filePath = "openapi.yaml"
+targetModule = "delivery"
+```
+- Introduced the `add` sub-command to the OpenAPI tool to update the Ballerina.toml file with the OpenAPI tool configuration details.
+Example:
+`bal openapi add -i <yaml file> --mode client --id <tool config id>`
+- Added OpenAPI mapping for the Ballerina constraints in OpenAPI specification generation.
+- Added OpenAPI link field mapping for the Ballerina HATEOAS feature in OpenAPI specification generation.
+- Added OpenAPI mapping for the Ballerina HTTP interceptor services in OpenAPI specification generation.
+- Added OpenAPI response mapping for the Ballerina HTTP status code errors in OpenAPI specification generation.
+- Added the support for the Ballerina client generation with status code response binding, this can be enabled by providing the `--with-status-code-binding` flag to the `openapi` client generation command.
```

Comment:
```suggestion
- Added support for the Ballerina client generation with status code response binding. This can be enabled by providing the `--with-status-code-binding` option to the OpenAPI client generation command.
```